src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/analysis/AnalysisDetailsPage.java

```diff
@@ -48,10 +48,10 @@
 	@FindBy(id="t-download-all-files-btn")
 	private List<WebElement> downloadAllFilesButton;
 
-	@FindBy(className="ant-dropdown-trigger")
+	@FindBy(css="#t-download-all-files-btn button.ant-dropdown-trigger")
```

Comment:
Don't use id's for test classes (I don't know if this was you, but can you update it to be a class please)

Reply:
Please see 5723b6770ef51e6c47300c825a9317cb05d96464.

Reply:
Please see dcf9d371514ce86754513908d8bc8ab29e0fecbf